acceptance/tests/resource/service/ticket_4124_should_list_all_disabled.rb

```diff
@@ -1,12 +1,10 @@
 test_name "#4124: should list all disabled services on Redhat/CentOS"
+confine :to, :platform => /el-5/
```

Comment:
Pragmatically, 'el-5' covers all of the 'el' derivatives, since the `:platform` defines the OS apis for Beaker not necessarily the OS which is normally expressed with the `:template` that is used to deploy the VM. However, all are part of the supported whitelist for Beaker supported platforms. So, I agree that it make sense to be explicit here.

Thanks @whopper !